tailwind.config.js

```diff
@@ -175,6 +175,7 @@ module.exports = {
       wider: '.02em' // 2%
     },
     fontSize: {
+      '2xs': '.625rem', // 10px
```

Comment:
```suggestion
      'xxs': '.625rem', // 10px
```

more consistent nah ?

Reply:
```
fontSize: {
      '2xs': '.625rem', // 10px
      xs: '.75rem', // 12px
      sm: '.875rem', // 14px
      base: '1rem', // 16px
      lg: '1.125rem', // 18px
      xl: '1.25rem', // 20px
      '2xl': '1.5rem', // 24px
      '3xl': '1.875rem', // 30px
      '4xl': '2.25rem', // 36px
      '5xl': '2.5rem', // 40px
      '6xl': '3.75rem' // 60px
    }
```

This is what I call consistency 🤔